src/test/resources/shared-params.xml

```diff
@@ -0,0 +1,555 @@
+        </memberClass>
+        <memberCode>1710128-9</memberCode>
+        <name>Gofore</name>
+        <subsystem id="id1">
+            <subsystemCode>Management</subsystemCode>
+        </subsystem>
+        <subsystem id="id2">
+            <subsystemCode>SS1</subsystemCode>
+        </subsystem>
+        <subsystem id="id3">
+            <subsystemCode>JANNEN</subsystemCode>
+        </subsystem>
+        <subsystem id="id4">
+            <subsystemCode>MANSIKKA</subsystemCode>
+        </subsystem>
+        <subsystem id="id5">
+            <subsystemCode>KIRSIKKA</subsystemCode>
+        </subsystem>
+        <subsystem id="id6">
+            <subsystemCode>HERUKKA</subsystemCode>
+        </subsystem>
+        <subsystem id="id7">
+            <subsystemCode>VAMMA/MARJA</subsystemCode>
+        </subsystem>
+        <subsystem id="id8">
+            <subsystemCode>Catalog</subsystemCode>
+        </subsystem>
+        <subsystem id="id9">
+            <subsystemCode>SS-cataloglister-testaamiseen</subsystemCode>
+        </subsystem>
+        <subsystem id="id10">
+            <subsystemCode>MANAGEMENT</subsystemCode>
+        </subsystem>
+    </member>
+    <member id="id11">
+        <memberClass>
+            <code>GOV</code>
+            <description>Governmental</description>
+        </memberClass>
+        <memberCode>1945065-0</memberCode>
+        <name>Vaakon Nakki</name>
+        <subsystem id="id12">
+            <subsystemCode>VAAKKO1</subsystemCode>
+        </subsystem>
+        <subsystem id="id13">
+            <subsystemCode>REST-GW-CONSUMERS</subsystemCode>
+        </subsystem>
+        <subsystem id="id14">
+            <subsystemCode>MYSERVICE</subsystemCode>
+        </subsystem>
+    </member>
+    <member id="id15">
+        <memberClass>
+            <code>GOV</code>
+            <description>Governmental</description>
+        </memberClass>
+        <memberCode>15334513</memberCode>
+        <name>Pepe Design</name>
+        <subsystem id="id16">
+            <subsystemCode>SUBS</subsystemCode>
+        </subsystem>
+    </member>
+    <member id="id17">
+        <memberClass>
+            <code>GOV</code>
+            <description>Governmental</description>
+        </memberClass>
+        <memberCode>13775550</memberCode>
+        <name>Kampaamo Lissu</name>
+        <subsystem id="id18">
+            <subsystemCode>SUBS</subsystemCode>
+        </subsystem>
+    </member>
+    <member id="id19">
+        <memberClass>
+            <code>GOV</code>
+            <description>Governmental</description>
+        </memberClass>
+        <memberCode>88834888</memberCode>
+        <name>Ritvan Kala-Kahvila</name>
+    </member>
+    <member id="id20">
+        <memberClass>
+            <code>GOV</code>
+            <description>Governmental</description>
+        </memberClass>
+        <memberCode>PWNED&lt;script&gt;console.log("OWNED!")&lt;/script&gt;</memberCode>
```

Comment:
would be good to sanitize the test data to not contain actual real life names (https://pv6ttsa01.csc.fi), real hashes etc, and also to keep to file size not bigger than is needed.